<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Du Lieu Ca Kieng</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="sideArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>651</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="sideWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>239</width>
       <height>649</height>
      </rect>
     </property>
     <widget class="QTreeView" name="categories">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="mainArea">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>574</width>
      <height>651</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="mainWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>572</width>
       <height>649</height>
      </rect>
     </property>
     <widget class="QTableView" name="productData">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>572</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="noneData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vui lòng chọn danh mục</string>
      </property>
     </widget>
     <zorder>noneData</zorder>
     <zorder>productData</zorder>
    </widget>
   </widget>
   <widget class="QScrollArea" name="detailsArea">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>496</width>
      <height>651</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="detailsWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>494</width>
       <height>649</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>230</y>
        <width>483</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionLabel">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>200</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mô tả</string>
      </property>
     </widget>
     <widget class="QLabel" name="img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>225</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{border: 1px solid black;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="activeImg">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>80</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Dùng ảnh</string>
      </property>
      <widget class="QRadioButton" name="prodImg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Riêng</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="catImg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Chung</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="briefImgScale">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>80</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Ảnh tóm tắt</string>
      </property>
      <widget class="QRadioButton" name="briefFull">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>100%</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="briefHalf">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>50%</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="briefQuarter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>25%</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="briefMin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>12.5%</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="dropBriefImg">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>129</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cắt ảnh tóm tắt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ảnh tóm tắt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pasteDescription">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>82</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dán HTML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copyDescription">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>82</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Chép HTML</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="categoryArea">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>496</width>
      <height>651</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="categoryWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>494</width>
       <height>649</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>categoryArea</zorder>
   <zorder>sideArea</zorder>
   <zorder>mainArea</zorder>
   <zorder>detailsArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1345</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Dữ liệu</string>
    </property>
    <addaction name="actionData"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Lưu</string>
   </property>
  </action>
  <action name="actionMetaData">
   <property name="text">
    <string>&amp;Siêu dữ liệu</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>&amp;Danh sách</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionData</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchDataViewInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveAll()</slot>
  <slot>switchCharacteristicsView()</slot>
  <slot>switchDataViewInit()</slot>
 </slots>
</ui>
